python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -145,6 +145,16 @@ def aggregate(self, arg, split_every=None, split_out=1):
                 as_index=self.as_index,
             )
 
+            if self._slice:
+                result = result[self._slice]
+
+            if not isinstance(arg, (list, dict)) and isinstance(
+                result, DaskDataFrame
+            ):
+                result = result[result.columns[0]]
```

Comment:
Do we need to check that there is only one column here?

Reply:
Not sure, I'm mostly just replicating the post-processing that happens for a series groupby in Dask. I *think* this is a failsafe for the previous check, although that might not be needed if we know that `self._slice` is always non-null.

Looking at `CudfSeriesGroupby.mean()`, it looks like that might be the case, as there this step is done in the return call:

https://github.com/rapidsai/cudf/blob/8bd0dfe675249f3a831813745b8ebb5bef1519d4/python/dask_cudf/dask_cudf/groupby.py#L104-L114

So maybe just doing that for the general case is sufficient.